rep-I0004.rst

```diff
@@ -286,12 +286,16 @@ Motoman
 
       2001  MOTOMAN_MOTION_CTRL                   -
       2002  MOTOMAN_MOTION_REPLY                  -
-      2003  MOTOMAN_READ_SINGLE_IO                -
-      2004  MOTOMAN_READ_SINGLE_IO_REPLY          -
-      2005  MOTOMAN_WRITE_SINGLE_IO               -
-      2006  MOTOMAN_WRITE_SINGLE_IO_REPLY         -
+      2003  ROS_MSG_MOTO_READ_IO_BIT              -
+      2004  ROS_MSG_MOTO_READ_IO_BIT_REPLY        -
+      2005  ROS_MSG_MOTO_WRITE_IO_BIT             -
+      2006  ROS_MSG_MOTO_WRITE_IO_BIT_REPLY       -
```

Comment:
'Officially' changing the name associated with an already registered message identifier is something that would need to be voted on, but we can probably make an exception as these are in the Motoman-specific range.